src/Npgsql/Npgsql.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+	<PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.0" />
```

Comment:
Please let's not add a dependency just for this one hash code calculation...

Reply:
I think, it makes sense to remove .NET Standard 2 next time.

Reply:
I don't think we should remove TFMs because of something like this...

Reply:
It's pretty old or will be one year later and there's .NET Standard 2.1.

Reply:
.NET Standard 2.1 doesn't support .net framework

Reply:
I hope it will be dead in one year...